src/server/controllers/signInControllers.ts

```diff
@@ -400,13 +400,11 @@ export const oktaIdxApiSignInController = async ({
 	// get the email and password from the request body
 	const { email = '', password = '', passcode } = req.body;
 
-	const usePasscodeSignInFlag =
-		res.locals.queryParams.usePasscodeSignIn ||
-		res.locals.abTestAPI.isUserInVariant('PasscodeSignInTest', 'variant');
+	// we're using passwords if the `usePasswordSignIn` query parameter is set
+	const usePasscodeSignInFlag = !res.locals.queryParams.usePasswordSignIn;
 
 	try {
-		// only attempt to sign in with a passcode if the user currently has the query parameter set
-		// this should be removed when we're ready to enable this for all users
+		// only attempt to sign in with a passcode if we've enabled passcodes for all users
```

Comment:
is this comment correct? it seems like this variable means that the user has not forced password sign in with the parameter, right ?

Reply:
I've updated the comments on this page to make it clearer about whats going on!